Scroller/Scroller.js

```diff
@@ -45,8 +45,39 @@ const nop = () => {};
  * @public
  */
 let Scroller = (props) => {
-	// Hooks
+	const {
```

Comment:
Could you please check changes of Scroller and VirtualList in https://github.com/enactjs/sandstone/pull/1613/files?
With the current implementation we cannot load defaultProp values properly in Scroller sampler.